src/Chapter2.hs

```diff
@@ -336,7 +347,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList l r xs =
+  if l<0 || r<0 || l>r then []
```

Comment:
In Haskell, functions and operators are always separated from arguments with spaces 👾 

```suggestion
  if l < 0 || r < 0 || l > r then []
```